build_tools/buildkite/cmake/android/benchmark/pipeline.yml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 Google LLC
```

Comment:
So given that we have a directory arm64-v8a and this also tests on that architecture, it seems a bit weird that this isn't under that directory. Maybe it will turn out that that directory structure won't be useful (I copied it from what we set up for Kokoro, which has different requirements), but while we have it, let's add this there too. Concretely, I think my suggestion is

```shell
mv build_tools/buildkite/cmake/android/benchmark/pipeline.yml build_tools/buildkite/cmake/android/arm64-v8a/benchmark.yml
```